README.md

```diff
@@ -56,6 +56,7 @@ module.exports = {
 |:--:|:-----:|:----------|
 |**`literate`**|`false`|Enable CoffeeScript in Markdown (Code Blocks) e.g `file.coffee.md`|
 |**`sourceMap`**|`false`|Enable/Disable Sourcemaps|
+|**`transpile`**|`false`|Provide Babel presets and plugins|

```

Comment:
Coffeescript uses babel under the hood by default or are other transpilers possible in theory ?

Reply:
Yeah, there is a built in support only for Babel as I see.